datastore/db/bytebase/FertiScan/schema_0.0.13.sql

```diff
@@ -0,0 +1,323 @@
+    "id" uuid NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    "picture_set" json NOT NULL,
+    "owner_id" uuid NOT NULL REFERENCES "fertiscan_0.0.13".users(id),
+    "upload_date" date NOT NULL DEFAULT current_timestamp,
+    "name" text
+    );
+        
+    alter table "fertiscan_0.0.13".users ADD "default_set_id" uuid REFERENCES "fertiscan_0.0.13".picture_set(id);
+
+    CREATE TABLE "fertiscan_0.0.13"."picture" (
+    "id" uuid NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    "picture" json NOT NULL,
+    "nb_obj" int,
+    "picture_set_id" uuid NOT NULL REFERENCES "fertiscan_0.0.13".picture_set(id),
+    "verified" boolean NOT NULL DEFAULT false,
+    "upload_date" TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."province" (
+    "id" SERIAL PRIMARY KEY,
+    "name" text UNIQUE NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."region" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "province_id" int REFERENCES "fertiscan_0.0.13".province(id),
+    "name" text NOT NULL
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."location" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "name" text,
+    "address" text NOT NULL,
+    "region_id" uuid References "fertiscan_0.0.13".region(id)
+    );    
+    
+    CREATE TABLE "fertiscan_0.0.13"."organization_information" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "name" text,
+        "website" text,
+        "phone_number" text,
+        "location_id" uuid REFERENCES "fertiscan_0.0.13".location(id),
+        "edited" boolean DEFAULT false
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."organization" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "information_id" uuid REFERENCES "fertiscan_0.0.13".organization_information(id),
+    "main_location_id" uuid REFERENCES "fertiscan_0.0.13".location(id)
+    );
+
+
+    Alter table "fertiscan_0.0.13".location ADD "owner_id" uuid REFERENCES "fertiscan_0.0.13".organization(id);
+
+    CREATE TABLE "fertiscan_0.0.13"."sample" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "number" uuid,
+    "collection_date" date,
+    "location" uuid REFERENCES "fertiscan_0.0.13".location(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."unit" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "unit" text NOT NULL,
+    "to_si_unit" float
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."element_compound" (
+    "id" SERIAL PRIMARY KEY,
+    "number" int NOT NULL,
+    "name_fr" text NOT NULL,
+    "name_en" text NOT NULL,
+    "symbol" text NOT NULL UNIQUE
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."label_information" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "product_name" text,
+    "lot_number" text,
+    "npk" text,
+    "registration_number" text,
+    "n" float,
+    "p" float,
+    "k" float,
+    "guaranteed_title" text,
+    "guaranteed_titre" text,
```

Comment:
I can switch it up to stick to our convention.

Since it seems like a already long key, I wanted to shorten it out by using titre/title to differenciate the language

Reply:
I suggest we don't delay these changes.